cashu/wallet/settings.py

```diff
@@ -0,0 +1,49 @@
+from pydantic import BaseSettings, AnyUrl, Field, root_validator, SecretStr
```

Comment:
Unrelated to PR?

Reply:
Nice use of `BaseSettings` though, I think we should keep it. Does it play along with the rest of the software though?

Reply:
fair - i'll remove for now. would you find it useful to use this instead of the other `cashu/core/settings.py` ?